appeals/web/src/server/lib/mappers/data/appeal/submappers/lpa-statement.mapper.js

```diff
@@ -0,0 +1,20 @@
+import { dateISOStringToDisplayDate } from '#lib/dates.js';
+import { mapDocumentStatus } from '#lib/display-page-formatter.js';
+import { documentationFolderTableItem } from '#lib/mappers/index.js';
+
+/** @type {import('../mapper.js').SubMapper} */
+export const mapLpaStatement = ({ appealDetails, currentRoute }) => {
+	return documentationFolderTableItem({
+		id: 'lpa-statement',
+		text: 'LPA statement',
+		statusText: mapDocumentStatus(appealDetails?.documentationSummary?.lpaStatement?.status),
+		receivedText: dateISOStringToDisplayDate(
+			appealDetails?.documentationSummary?.lpaStatement?.receivedAt instanceof Date
+				? appealDetails?.documentationSummary?.lpaStatement?.receivedAt.toDateString()
+				: appealDetails?.documentationSummary?.lpaStatement?.receivedAt
```

Comment:
The API should always serialise `receivedAt` into a string so I think this will always return false.

Reply:
Yeah I didn't really understand why the type system required this coercion but it's certain this might be a Date